data-prepper-plugins/http-source/build.gradle

```diff
@@ -10,8 +10,10 @@ plugins {
 dependencies {
     implementation project(':data-prepper-api')
     implementation project(':data-prepper-plugins:blocking-buffer')
+    implementation project(':data-prepper-plugins:http-source-common')
     implementation project(':data-prepper-plugins:common')
     implementation project(':data-prepper-plugins:armeria-common')
+    implementation project(':data-prepper-test-common')
```

Comment:
Please make this `testImplementation` and move to set with the other `testImplementation` dependencies a few lines below.